packages/components/src/components/time-picker/time-picker.tsx

```diff
@@ -596,7 +596,7 @@ export class TimePicker extends LitElement implements TimeComponent {
         )}
         {showSecondSuffix && (
           <span class={{ [CSS.delimiter]: true, [CSS.secondSuffix]: true }}>
-            {localizedSecondSuffix}
+            {localizedSecondSuffix.trim()}
```

Comment:
Should this not be trimmed to honor all literals from `Intl.DateTimeFormat#formatToParts`?

Reply:
It seems very minor, but due to the way the `calcite-time-picker` is designed, the padding between the values accommodates any whitespace that would be provided by the browser via literals, so if we don't trim it here, it adds extra unnecessary space as shown in this Chromatic diff: 
<img width="289" height="173" alt="image" src="https://github.com/user-attachments/assets/3224b90c-ceac-442f-99cc-8eb2ccc701c8" />

Unless we redesign `calcite-time-picker` (cc @ashetland ) to more closely align to the literals that the browser is returning for the time format, I think it makes sense to (at least for now) trim in this specific instance to keep the gap between the seconds and the meridiem from being unnecessarily larger.

Reply:
I agree. If these changes are making the picker more spread out, we should try to mitigate that. That picker is due for a post-masked-input redesign, but a full rethink is out-of-scope for now.

Reply:
Oh wait, @jcfranco are you suggesting trimming the extra space from all? If that would make `calcite-time-picker` have less white space, then I'm for that.

Reply:
No, I was suggesting to honor the spacing that we get from Intl when it parses the date format parts. For `fr-CA`, it included some spacing that was unintentionally being removed.